docs/src/main/sphinx/release/release-431.md

```diff
@@ -0,0 +1,55 @@
+# Release 431 (25 Oct 2023)
+
+## General
+
+* Add support for `CREATE OR REPLACE TABLE` statement. ({issue}`13180`)
+* Disallow a null offset in {func}`lead` and {func}`lag`. ({issue}`19003`)
+
+## Web UI
+
+* Mask shared secret value in logs. ({issue}`19519`)
+
+## JDBC driver
+
+* Add a new JDBC property `legacyPreparedStatements` that allows to control
+  whether `EXECUTE IMMEDIATE` is used to execute prepared statements. ({issue}`18957`)
+
+## Delta Lake connector
+
+* Replace the Hive metastore configuration property `hive.metastore-timeout`
+  with the `hive.metastore.thrift.client.connect-timeout` and
+  `hive.metastore.thrift.client.read-timeout` properties. ({issue}`19390`)
+
+## Hive connector
+
+* Replace the Hive metastore configuration property `hive.metastore-timeout`
+  with the `hive.metastore.thrift.client.connect-timeout` and
+  `hive.metastore.thrift.client.read-timeout` properties. ({issue}`19390`)
+* Mask Hive S3 proxy password value in logs. ({issue}`19519`)
```

Comment:
Do these two have to go into Iceberg, Delta Lake and Hudi as well @wendigo ?

Reply:
I'm pretty sure they are used there also... the file system code is shared across all of them